stories/atoms/components/textField/TextField.stories.mdx

```diff
@@ -266,13 +266,22 @@ The `withBorder` property allows you to add a themed border to the textField and
 
 ## Multiline
 
-The `multiline` property allows you to write text on several lines. The simple text field becomes a text area.  
-When `multiline` is `true` and [disableAreaResize](/docs/atoms-textfield--disable-area-resize) is `false` the [size](/docs/atoms-textfield--sizes) is automatically `auto`.
+The `multiline` property allows you to write text on several lines. The simple text field becomes a text area.
+The [`size`](/docs/atoms-textfield--sizes) is automatically **auto**.  
+When `fullWidth` is **true** the user can only resize the height of the textField.
```

Comment:
Maybe you could rework this description to have fewer very short sentences and a more airy text with possibly some line breaks?